### ISSUE TITLE
The 401 - Authentification Error 

### ISSUE THREAD

**Issue by duttakapil** [coderplex/learnplex#109]
This error keeps flashing on the screen of the user randomly through out their entire experience of the website. It shows up when they delete a page and the site reloads, when they switch between pages sometimes

**Comment by pbteja1998** [comment]
@duttakapil This should be fixed. Check and see if you can reproduce this anywhere now.

**Comment by duttakapil** [comment]
Yeah, I do not notice this right now. If I do, will reopen the issue
